.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -36,7 +36,7 @@ Note: remove the backticks (``) from the link.
 - [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
 
 <!--
-If this release will become the next LTS release, add/update the checkboxes above as needed e.g.
+If this release will become the next LTS release (ie. if the Monday of this week was the last Monday of the month), add/update the checkboxes above as needed e.g.
```

Comment:
VERY open to suggestions on how to concisely communicate this. I could link to other LTS docs, but I feel like it would be better to just explain this clearly here (since the other LTS docs explain how to determine when the LTS promote happens, but not how to tell if the release currently being cut will end up as LTS)

Reply:
I've adopted this suggestion, but would love to hear from the rest of the reviewers if they have any suggestions for making it even clearer

Reply:
I like @rsimha's suggestion! One note is that releases are pushed to stable on Wednesdays, not Mondays.

Also I wouldn't say `it will never be promoted to LTS` but something along the lines of `it will not make this month's LTS cut` or `it is not an LTS release candidate` because those PRs will eventually make it into next month's LTS.

Reply:
Added clarification that it's included in a later LTS release as well; PTAL